translations/pencil_fr.ts

```diff
@@ -47,12 +47,12 @@
     <message>
         <location filename="../app/src/actioncommands.cpp" line="133"/>
         <source>No sound layer exists as a destination for your import. Create a new sound layer?</source>
-        <translation>Votre piste son ne peut pas être importée car aucune piste de son n&apos;existe. Créer une nouvelle piste de son ?</translation>
+        <translation>Votre couche sonore ne peut pas être importée car aucune couche sonore n&apos;existe. Créer une nouvelle couche sonore ?</translation>
```

Comment:
This warning only comes up when trying to import a individual sound and no sound layer exists to add that sound to. Describing what's being imported as a sound layer is confusing and probably wrong.